code/__HELPERS/game.dm

```diff
@@ -298,7 +298,11 @@
 			if(G.client != null)
 				if(!(G.mind && G.mind.current && G.mind.current.stat != DEAD))
 					if(!G.client.is_afk(afk_bracket) && (G.client.prefs.be_special & be_special_flag))
-						candidates += G.client
+						if (jobbanType)
+							if(!(jobban_isbanned(G, jobbanType) || jobban_isbanned(G, "Syndicate")))
+								candidates += G.client

```

Comment:
IIRC jobbanning someone from Syndicate is how the system handles banning someone from all antag roles at once.